apps/api/src/app/endpoints/ai/ai.service.ts

```diff
@@ -10,7 +10,19 @@ import type { AiPromptMode } from '@ghostfolio/common/types';
 import { Injectable } from '@nestjs/common';
 import { createOpenRouter } from '@openrouter/ai-sdk-provider';
 import { generateText } from 'ai';
-import type { ColumnDescriptor } from 'tablemark';
+import { ColumnDescriptor } from 'tablemark';
+
+const tablemark = require('tablemark').default;
+
+// Column name constants for holdings table
+const HOLDINGS_TABLE_COLUMNS = {
```

Comment:
This is a good start.

Is it possible to create an array out of it with type `({key: string }&ColumnDescriptor)[]`:
```
[
  { key: 'CURRENCY', name: 'Currency' },
  ...
  { align: 'right', key: 'ALLOCATION_IN_PERCENTAGE', name: 'Allocation in Percentage' },
  ...
]
```